docs/source/guide/api.md

```diff
@@ -39,27 +45,101 @@ label-studio user --username <username>
+
+#### Create and set up a project
 Create a project and set up the labeling interface in Label Studio using the API. See the [Create new project API endpoint documentation](/api#operation/api_projects_create).
 
 If you want to make sure the configuration for your labeling interface is valid before submitting it using the API, you can use the [validate label config](/api#operation/api_projects_validate_create) API endpoint.
 
-### Import tasks using the API
+### Storage
+You can add source storage connections to sync data from an external source to a Label Studio project, and add target storage connections to sync annotations from Label Studio to external storage.
 
-To import tasks using the API, make sure you know the project ID that you want to add tasks to. See additional examples and parameter descriptions in the [import data endpoint documentation](/api#operation/api_projects_import_create)
+For more information on the types of cloud storage, read the following:
+
+1. [Microsoft Azure storage](storage.html#Microsoft-Azure-Blob-storage) 
+
+2. [Google Cloud storage](storage.html#Google-Cloud-Storage)
 
-### Retrieve tasks
+3. [Local storage](storage.html#Local-storage)
+
+4. [Redis storage](storage.html#Redis-database)
+
+5. [Amazon S3](storage.html#Amazon-S3)
+
+Also, read the API documentation for [Microsoft Azure storage](/api#tag/Storage:-Azure), [Google Cloud storage](/api#tag/Storage:-GCS), [Local storage](/api#tag/Storage:-Local), [Redis storage](/api#tag/Storage:-Redis), and [Amazon S3](/api#tag/Storage:S3). 
+
+### Tasks
+A task represents a distinct item from a dataset that is ready to be labeled, pre-annotated, or has already been annotated. For example: a sentence of text, an image, or a video clip. 
+
+See [API documentation for Tasks](api#tag/Tasks).
+
+#### Retrieve tasks
 Retrieve a paginated list of tasks for a specific project. If you want, you can also retrieve tasks and annotations using this API endpoint, as an alternative to exporting annotations. See details and parameters in the [list project tasks endpoint documentation](/api#operation/api_projects_tasks_list).
 
-### Export annotations
+### Users
+[Users](signup.html) are people who signup and have access to use the Label Studio product and experience the various features. The currently logged in/authenticated user.
 
-To export annotations, first see [which formats are available to export for your project](/api#operation/api_projects_export_formats_read). 
+See [API documentation for Users](/api#tag/Users).
 
-Choose your selected format from the response and then call the export endpoint. See the [export annotations](/api#operation/api_projects_export_read) endpoint documentation for more details.
+### Webhooks
+A [webhook](/webhooks.html) in Label Studio allows you to set up integrations that subscribe to certain events that occur inside Label Studio. 
```

Comment:
For less tech savvy readers, can we add one more sentence:  
"They  act as an automated messages sent from Label-Studio when something occurs in the application. The event-driven communication is useful in triggering and managing automated activity executed by external APIs"